wallettool/build.gradle

```diff
@@ -4,7 +4,7 @@ plugins {
     id 'java'
     id 'application'
     id 'org.asciidoctor.jvm.convert' version '3.3.2' apply false
-    id 'org.graalvm.buildtools.native' version '0.9.27' apply false
+    id 'org.graalvm.buildtools.native' version '0.9.28' apply false
```

Comment:
I think we can (and should) use version 0.10.2 here. See my comment on the main thread.

Reply:
Ok, I changed it to 0.10.2.